python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -352,11 +354,12 @@ def _read_master_cache(self):
                 with open(_master_idx, 'r') as input_f:
                     this_master_cache = pickle_from_file(input_f)[0]
                 for this_cache in this_master_cache:
-                    this_id = this_cache[0]
-                    self._cache_load_timestamp[this_id] = this_cache[1]
-                    self._cached_cat[this_id] = this_cache[2]
-                    self._cached_cls[this_id] = this_cache[3]
-                    self._cached_obj[this_id] = this_cache[4]
+                    if this_cache[4] >= 0:
```

Comment:
That is the time, and yes it is potentially -ve if something went wrong in a previous version of ganga were bad caches were still saved.